src/registry/docker/docker_test.go

```diff
@@ -0,0 +1,27 @@
+package docker_test
+
+import (
+	"testing"
+
+	"github.com/newrelic-forks/reroller/src/registry/docker"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Dockerhub(t *testing.T) {
+	infofunc := docker.DockerLikeImageInfo(docker.BaseUrl)
+
+	_, err := infofunc("bitnami/nginx", "latest")
+	require.NoError(t, err)
+
+	digests, err := infofunc("newrelic/infrastructure-bundle", "1.0.0")
+	require.NoError(t, err)
+	require.Contains(t, digests, "sha256:7300aec653dbe8aaef96b92d2f8319a2cef8e03e99b4420ba846b46a1447ea6b")
+
+	// test buildkit image https://github.com/moby/buildkit/issues/2251
```

Comment:
Could we split this to a second test? That way we will know which one is failing.